build/pause/Makefile

```diff
@@ -60,8 +58,10 @@ all: all-container
 
 all-push: all-push-images push-manifest
 
-push-manifest: manifest-tool
-	manifest-tool push from-args --platforms $(call join_platforms,$(ALL_ARCH)) --template $(IMAGE)-ARCH:$(TAG) --target $(IMAGE):$(TAG)
+push-manifest:
+	docker manifest create --amend $(IMAGE):$(TAG) $(shell echo $(ALL_ARCH) | sed -e "s~[^ ]*~$(IMAGE)\-&:$(TAG)~g")
+	set -x; for arch in $(ALL_ARCH); do docker manifest annotate --arch $${arch} ${IMAGE}:${TAG} ${IMAGE}-$${arch}:${TAG}; done
```

Comment:
we usually have this line as `@` but then you can't see the manifest actually being built, instead I removed that and added trace mode so you can see the annotations ...

Reply:
this is otherwise from build/debian-base

Reply:
Is this rule standardized? Should this makerule go into `../../hack/make-rules/Makefile.manifest`?

Reply:
it's used by a few things.

I'd somewhat prefer to finish getting the pause update sorted before refactoring further but I _think_ it should be pretty simple to standardize a few makefiles on this.